File: LV1/1.5/mixedfractions/mixedfractions.go
package main

import "fmt"

func main() {
	var a, b int
	for {
		fmt.Scanf("%d %d", &a, &b)
		if a == 0 && b == 0 {
			break
		}

		fmt.Println(a/b, a%b, "/", b)
	}
}

File: LV1/1.5/mixedfractions/mixedfractions.cpp
#include <cstdio>

int main() {
    int a, b;

    while (true) {
        scanf("%d %d", &a, &b);
        if (a == 0 && b == 0) {
            break;
        }
        printf("%d %d / %d\n", a / b, a % b, b);
    }
}